skills/cairo-auditor/tests/validate_deep_smoke.py

```
@@ -76,6 +83,358 @@ def run_fixture_scan() -> tuple[bool, str]:
+            return False, f"adversarial fixture preflight exited {proc.returncode}: {proc.stderr.strip()}"
+        try:
+            payload = json.loads(proc.stdout.strip())
+        except json.JSONDecodeError as exc:
+            return False, f"adversarial fixture preflight stdout is not JSON: {exc}"
+        findings = int(payload.get("findings", -1))
+        if findings != 0:
+            return False, f"expected adversarial fixture to have 0 deterministic findings, got {findings}"
+    return True, "adversarial fixture is not caught by deterministic preflight"
+
+
+def run_structured_deep_path() -> tuple[bool, str]:
```

Ruff (0.15.12)

[warning] Too many return statements (7 > 6)

(PLR0911)